api/auth/serialization.test.js

```diff
@@ -21,16 +19,14 @@ tap.test('passport serialization', async serializationTest => {
     done();
   });
 
-  serializationTest.test('serialize a user', async serializeTest => {
+  serializationTest.test('serialize a user', serializeTest => {
     const user = { id: 'the-user-id' };
     sessionStore.addSession.resolves('session-id');
 
-    await serialization.serializeUser(user, doneCallback, { sessionStore });
-
-    serializeTest.ok(
-      doneCallback.calledWith(null, 'session-id'),
-      'serializes the user object'
-    );
+    serialization.serializeUser(user, { sessionStore }).then(sessionId => {
+      serializeTest.same(sessionId, 'session-id', 'serializes the user object');
+      serializeTest.end();
+    });
```

Comment:
I'm curious about why this was changed from an `await`-type promise to using a `then`, just as a matter of consistency.

Reply:
I was on the fence about this but I did it first and was favoring the `then` type.  The second one needed the `await` syntax so you're right we should make them consistent.